src/services/ansiblePlaybook.ts

```diff
@@ -0,0 +1,179 @@
+import * as child_process from 'child_process';
+import { ExecException } from 'child_process';
```

Comment:
Why do you have two imports like this? Maybe just import what you need only?

Reply:
Thanks for pointing it out. That's a mistake caused due to auto imports in TS.